.github/workflows/docker_builds.yml

```diff
@@ -15,22 +15,22 @@ jobs:
         python_version: [3.6, 3.7, 3.8]
         pytorch_version: [1.1, 1.2, 1.3, 1.4, 1.5]
     steps:
-    - name: Extract Current Tag
-      if: contains(github.ref, 'refs/tags/')
-      id: get_version
-      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
     - uses: actions/checkout@v2
     - name: Publish Releases to Docker
       # only on releases
       uses: elgohr/Publish-Docker-Github-Action@2.14
-      if: contains(github.ref, 'refs/tags/') && !contains(${{ steps.get_version.outputs.VERSION }}, 'rc') %% !contains(${{ steps.get_version.outputs.VERSION }}, 'dev')
+      if: contains(github.ref, 'refs/tags/') && !contains(${{ steps.extract_branch.outputs.branch }}, 'rc') && !contains(${{ steps.extract_branch.outputs.branch }}, 'dev')
```

Comment:
branch is not RC, shall it be rather tag?

Reply:
Fixed by parsing both (we need branches for nightlies) and I thought tags are on the same ref, but they aren't. Good catch!